ingestion_program/run_ingestion.py

```diff
@@ -133,6 +133,12 @@
     sys.path.append(program_dir)
     sys.path.append(submission_dir)
 
+    for subdir in os.listdir(submission_dir):
+        sub_directory_path = os.path.join(submission_dir, subdir)
+        if os.path.isdir(sub_directory_path):
+            if not subdir.startswith("__"):
```

Comment:
These two `if`s can be combined? I think that will make it a little more readable.